pkg/controller/install/install_controller.go

```diff
@@ -135,14 +137,20 @@ func (r *ReconcileInstall) transform(instance *servingv1alpha1.Install) error {
 // Apply the embedded resources
 func (r *ReconcileInstall) install(instance *servingv1alpha1.Install) error {
 	// Ensure needed prerequisites are installed
-	for _, f := range platformPrereqFuncs {
+	for _, f := range platformPreInstallFuncs {
 		if err := f(r.client, r.scheme, instance.Spec.Namespace); err != nil {
 			return err
 		}
 	}
 	if err := r.config.ApplyAll(); err != nil {
 		return err
 	}
+	// Do any post-installation tasks
+	for _, f := range platformPostInstallFuncs {
+		if err := f(r.client, r.scheme, instance.Spec.Namespace); err != nil {
```

Comment:
I'm thinking of changing the signature for all the funcs to be `f(client, scheme, instance)` from which you can get `instance.GetNamespace()`

Reply:
I had the same thought as well, but didn't do it here as I know you were changing how namespace gets passed around anyway. I'd agree with doing what you said.